assets/js/pages/DatabasesOverview/DatabasesOverview.jsx

```diff
@@ -20,10 +20,10 @@ function DatabasesOverview({
   databases,
   databaseInstances,
   loading,
-  userAbilities,
   onTagAdd,
   onTagRemove,
   onInstanceCleanUp,
+  userAbilities,
```

Comment:
Why this change of order? :see_no_evil: 
Doesn't look better before all the `onX` functions?
Usually, we leave the function props at the end